doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -592,6 +592,13 @@ sdk-nrfxlib
 
 See the changelog for each library in the :doc:`nrfxlib documentation <nrfxlib:README>` for additional information.
 
+DFU target
+----------
+
+* Updated:
+  * Added new types :c:enum:`DFU_TARGET_IMAGE_TYPE_ANY_MODEM` and :c:enum:`DFU_TARGET_IMAGE_TYPE_ANY_APPLICATION` so downloading may accept any supported modem update type.
```

Comment:
```suggestion
  * Added new types :c:enum:`DFU_TARGET_IMAGE_TYPE_ANY_MODEM` and :c:enum:`DFU_TARGET_IMAGE_TYPE_ANY_APPLICATION`.
    This makes any supported modem update type acceptable when downloading.
```

Reply:
This was not done as I requested. All sentences must be on separate lines.

Reply:
I moved to one line, as there was some weird errors regarding indentation.